apps/examples/dtls_client/dtls_client_main.c

```diff
@@ -40,14 +40,32 @@
  * Included Files
  ****************************************************************************/
 
-#include "tls/config.h"
+#include <tinyara/config.h>
```

Comment:
You don't need to include config.h for this file. Because there is no config in this file.